<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="statek_name" default="$(eval open(find('statek_config') + '/name').read())"/>

    <!-- Launch twist to differential converter -->
    <group ns="$(arg statek_name)/converters">
        <node name="twist_to_differential" pkg="statek_hw" type="twist_to_differential_node.py">
            <param name="statek_name" type="string" value="$(arg statek_name)" />
            <rosparam command="load" file="$(find statek_config)/yaml/motion_config.yaml"/>
        </node>
    </group>

</launch>
